packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -116,8 +116,13 @@ describe('holocronModule', () => {
     const Module = holocronModule({
       name: 'mock-module',
       load,
-    })(({ moduleLoadStatus }) => <div>Mock Module - {moduleLoadStatus}</div>);
-    const mockStore = createStore(state => state);
+    })(({ moduleLoadStatus }) => (
+      <div>
+Mock Module -
```

Comment:
odd intentation, can this be pushed over? I'm guessing it affected a snapshot/test passing?

Reply:
similar for other instances in this file

Reply:
once the new version of eslint-config-amex is released i'll revert this changed